eagle-core/eagle-alert-parent/eagle-alert-app/src/main/resources/META-INF/providers/org.apache.eagle.alert.app.AlertUnitTopologyAppProvider.xml

```diff
@@ -25,6 +25,12 @@
     <configuration>
        <!-- alert topology sizing parameters -->
         <property>
+            <name>topology.name</name>
+            <displayName>topology.name</displayName>
+            <value>alertUnitTopology_1</value>
```

Comment:
I think we should not keep current implementation only supporting hard-coded "alertUnitTopology_1", and  coordinator should be able to easily fix it to use actual topology name accordingly.